src/components/ContributorIndicators.tsx

```diff
@@ -25,7 +25,7 @@ export const ContributorIndicators = ({
           {ticketView ? (
             <img src={OrcidLogo} height="20" alt="orcid" style={{ marginLeft: '0.2rem' }} />
           ) : (
-            <IconButton size="small" href={orcId} target="_blank">
+            <IconButton size="small" href={orcId} target="_blank" style={{ paddingRight: 0 }}>
```

Comment:
`paddingRight: 0` her gjør at selve knappen blir skeiv. Ingen veldig big deal, men ikke helt heldig at vi gjør det "by design" heller...
Hva med å sette `ml: "0.2rem"` på Box  over, og bare droppen padding her? Mister visning av hover for ORCID-knapp da, men det er kanskje ikke noe stort tap(?) 
```suggestion
            <IconButton size="small" href={orcId} target="_blank" style={{ padding: 0 }}>
```